src/ReverseProxy/Configuration/SessionAffinityConfig.cs

```diff
@@ -22,7 +22,7 @@ public sealed record SessionAffinityConfig
     public string? Policy { get; init; }
 
     /// <summary>
-    /// Strategy handling missing destination for an affinitized request.
+    /// Strategy handling missing destination for a request by affinity.
```

Comment:
```suggestion
    /// Strategy for handling a missing destination for an affinitized request.
```

"for a request by affinity" doesn't mean anything.